server/src/main/java/org/elasticsearch/monitor/metrics/NodeMetrics.java

```diff
@@ -134,7 +134,7 @@ private void registerAsyncMetrics(MeterRegistry registry) {
 
         metrics.add(
             registry.registerLongGauge(
-                "es.node.stats.indices.translog.uncommitted_operations",
+                "es.indices.translog.uncommitted_operations",
```

Comment:
we have `es.indices.translog.operations` and `es.indices.translog.uncomitted_operations`
should we group them?